packages/stores/src/queries/concentrated-liquidity/position-by-id.ts

```diff
@@ -144,6 +146,24 @@ export class ObservableQueryLiquidityPositionById extends ObservableChainQuery<{
     );
   }
 
+  /** Aggregation of claimable rewards by coin denom. */
+  @computed
+  get totalClaimableRewards(): CoinPretty[] {
+    return Array.from(
+      [...this.claimableSpreadRewards, ...this.claimableIncentiveRewards]
+        .reduce<Map<string, CoinPretty>>((sumByDenoms, coin) => {
+          const current = sumByDenoms.get(coin.currency.coinMinimalDenom);
```

Comment:
I think you could do 
`if (sumByDenoms.has(coin.currency.coinMinimalDenom))`

Might be clearer

Reply:
Yes but I'm going to need to get it to add to it on line 157 regardless?